packages/components/pie-input/test/component/pie-input.spec.ts

```diff
@@ -121,6 +121,21 @@ test.describe('PieInput - Component tests', () => {
                 expect((await input.inputValue())).toBe('');
             });
 
+            test('should default to an empty string if undefined value prop provided', async ({ mount }) => {
+                // Arrange
+                const component = await mount(PieInput, {
+                    props: {
+                        value: undefined,
+                    } as InputProps,
+                });
+
+                // Act
+                const input = component.locator('input');
+
+                // Assert
+                expect((await input.inputValue())).toBe('');
```

Comment:
Out of interest would something along the lines of `page.inputValue('pie-input')` work, or do you need to find the inner input?

Reply:
It's a nice suggestion. Unfortunately, Playwright throws on any element that isn't an input :

"Throws for non-input elements. However, if the element is inside the <label> element that has an associated control, returns the value of the control."

I would prefer to use an approach that doesn't check the underlying input though. I am unsure how we can do that. I considered something like:

`const inputValue = await page.evaluate(() => document.querySelector('pie-input')?.value);`

But it's cumbersome and didn't quite work either